src/api/.haml-lint_todo.yml

```diff
@@ -76,7 +76,6 @@ linters:
   SpaceBeforeScript:
     exclude:
       - "app/views/webui/search/owner.html.haml"
```

Comment:
Nitpick: It seems too that these [two comment lines](https://github.com/openSUSE/open-build-service/pull/5129/files#diff-25c39567a7540339900a516d996f6032R192) are not needed.